client/src/components/backToAllReviews.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const BackToAllReviews = props => (
```

Comment:
Include parentheses around props?

Reply:
I did at first and then eslint told be not too. It seems inconsistent when it wants me to or not, but I'm just following what it tells me. But good eye.

Reply:
That's interesting.  Okay cool.